pom.xml

```diff
@@ -104,18 +105,17 @@
                 <artifactId>validation-api</artifactId>
                 <version>2.0.1.Final</version>
             </dependency>
-            <dependency>
-                <groupId>com.fasterxml.jackson.core</groupId>
-                <artifactId>jackson-databind</artifactId>
-                <!-- CVE-2018-12022, CVE-2018-12023, CVE-2019-14379, CVE-2019-14439, CVE-2018-19362, CVE-2018-19361,
-                CVE-2019-12814, CVE-2018-19360, CVE-2018-11307, CVE-2018-14718, CVE-2018-1000873, CVE-2018-14719,
-                CVE-2018-14721, CVE-2018-14720, CVE-2019-12086, CVE-2019-12384 -->
-                <version>2.10.0.pr3</version>
-            </dependency>
```

Comment:
`jackson-databind` is pulled in by `openapi-generator-maven-plugin`, `jackson-databind-nullable`, and `spring-boot-starter-web`. `openapi-generator-maven-plugin:4.1.3` pulls in the newest version of `jackson-databind`, which fixes CVE-2019-16335 and CVE-2019-14540. Excluding `jackson-databind` from `jackson-databind-nullable` and `spring-boot-starter-web` means that only the newest version of `jackson-databind` is pulled in and that only one version of `jackson-databind` is pulled in.